tests/unit/test_cli/test_env_resolution.py

```diff
@@ -75,6 +78,8 @@ def test_env_from_env_autodiscover_from_files(
     dotted_path = _path_to_dotted_path(tmp_file_path.relative_to(Path.cwd()))
 
     assert isinstance(env.app, Litestar)
+
+    print("parent directory content: %s", list(tmp_file_path.parent.iterdir()))  # noqa: T201
```

Comment:
I'm trying to see what is in these tmp directories when this test fails.. I don't know any better way to do this unfortunately.